spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/services/impl/DefaultStreamServiceTests.java

```diff
@@ -126,6 +126,34 @@ public void verifyUpgradeStream() {
 	}
 
 	@Test
+	public void verifyUndeployStream() {
```

Comment:
Can we use he mocks setup in the `@Before`?

Reply:
Will fix this on merge.

Reply:
@cppwfs I thought this was fixed along with the other PR, no?